addons/account/views/account_menuitem.xml

```diff
@@ -65,7 +65,7 @@
                 <menuitem id="menu_action_tax_form" action="action_tax_form" sequence="2"/>
                 <menuitem id="menu_action_account_journal_form" action="action_account_journal_form" groups="account.group_account_manager" sequence="3"/>
                 <menuitem id="account_report_folder" name="Reporting" groups="account.group_account_readonly" sequence="4"/>
-                <menuitem id="menu_action_currency_form" action="base.action_currency_form" name="Currencies" groups="base.group_multi_currency" sequence="4"/>
+                <menuitem id="menu_action_currency_form" action="base.action_currency_form" name="Currencies" sequence="4"/>
```

Comment:
IMHO, there should still be a way to enable currencies in the settings, without having to go find an obscure account menuitem.

Reply:
ping @avw-odoo

Reply:
I have included a link to the currencies list now, so this should be improve accessibility